features.json

```diff
@@ -97,11 +97,16 @@
 		"Safari": {
 			"url": "https://www.apple.com/safari/",
 			"logo": "/images/safari_48x48.png",
-			"version": "13.1",
+			"version": "14.1",
 			"features": {
+				"bigInt": true,
+				"bulkMemory": "JSC_useWebAssemblyReferences, enabled in Safari Technology Preview",
```

Comment:
Wait is this flag correct? Why is `bulkMemory` under `JSC_useWebAssemblyReferences` and not `JSC_useWebAssemblyThreading` which it's normally grouped with?

Reply:
Let me check

Reply:
The flag is correct, bulk memory operations are under `JSC_useWebAssemblyReferences` flag in JSC

Reply:
That's... a weird coupling. Okay, thanks for checking.